static/panes/cfg-view.ts

```diff
@@ -22,427 +22,323 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+import {Pane} from './pane';
```

Comment:
Can't comment in unmodified files, but do you think we should change `views/templates/panes/compiler.pug:64` to `Control Flow Graph`?